tests/unit/test_column_noise.py

```diff
@@ -599,14 +669,36 @@ def test_write_wrong_digits(dummy_dataset):
 
     # Check expected noise level
     check_original = data[~missing_mask]
+    # Remember our data looks like this
```

Comment:
Should we enforce more directly that this is what our data looks like? For example, storing this list as a constant?

Reply:
They are stored at the top where we build up the dummy_data fixture.